runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -199,7 +203,7 @@ public void onVarbitChanged(VarbitChanged event)
 		{
 			int lastSackValue = curSackSize;
 			refreshSackValues();
-			shouldUpdateOres = curSackSize != lastSackValue;
+			shouldUpdateOres = curSackSize < lastSackValue;
```

Comment:
Wouldn't you just have to create the snapshot here? And all it really has to do is keep track of each ore type and not take a snapshot of the whole inventory.

Reply:
I suppose it might be a bit more efficient to take the snapshot here but the snapshot I take only counts the ores anyways, it doesn't track any non ore/nugget items.